jvm/workbookapp/src/main/kotlin/org/wycliffeassociates/otter/jvm/workbookapp/ui/screens/WorkbookPage.kt

```diff
@@ -78,6 +79,8 @@ class WorkbookPage : View() {
     private var exportProgressListener: ChangeListener<Boolean>? = null
     private val tabChaptersListeners = mutableMapOf<String, ListChangeListener<ChapterCardModel>>()
 
+    private val tabHeightProperty = SimpleDoubleProperty()
```

Comment:
This doesn't look right to me when we need a dedicated class property to implement UI sizing